transforms/angle-brackets/transform.js

```diff
@@ -263,11 +263,15 @@ function transformNodeAttributes(tagName, node) {
 }
 
 function getDataAttributesFromParams(params) {
-  return params.filter(param => param.original && `${param.original}`.startsWith('data-'));
+  return params.filter(
+    p => p.original && `${p.original}`.startsWith('data-') && p.type === 'PathExpression'
```

Comment:
This can't be simplified to just:

```ts
p => p.type === 'PathExpression'
```
So being explicit with `p.type === 'PathExpression'` is I think the only improvement